spec/tags/truffle/launcher_tags.txt

```diff
@@ -13,3 +13,11 @@ slow:The launcher prioritises options on the command line over system properties
 slow:The launcher prioritises options on the command line using -X over system properties
 slow:The launcher allows -cp in JAVA_OPTS
 slow:The launcher allows -classpath in JAVA_OPTS
+aot:The launcher prints the full java command with -J-cmd
+aot:The launcher prints the full java command with -cmd in JAVA_OPTS
+aot:The launcher adds options from $JAVA_OPTS to the command
+aot:The launcher preserve spaces in options
```

Comment:
It would be nice to have a variant of this spec for AOT (just without the leading -J)